client/landing/gutenboarding/components/plans/plans-grid/index.tsx

```diff
@@ -68,10 +69,20 @@ const PlansGrid: React.FunctionComponent< Props > = ( {
 
 			<div className="plans-grid__details">
 				{ showDetails && (
-					<div className="plans-grid__details-heading">
-						<Title>{ __( 'Detailed comparison' ) }</Title>
-						<PlansDetails />
-					</div>
+					<Spring
+						from={ { maxHeight: '0px' } }
+						to={ { maxHeight: '2000px' } }
```

Comment:
Fixed to height seems odd and I could imagine this depends on locales (lengths of strings) and over time on feature list we get from store.